packages/data-point-service/lib/revalidation-store.js

```diff
@@ -0,0 +1,85 @@
+const debug = require('debug')('data-point-service:cache')
+const throttle = require('lodash/throttle')
+
+const MAX_STORE_SIZE = 10000
+const THROTTLE_WAIT = 1000 // cleanup in every 1 seconds
+
+/**
+ * It marks and deletes all the keys that have their ttl expired
+ * @param {Map} store Map Object that stores all the cached keys
+ */
+function clear (store) {
+  const forDeletion = []
+  const now = Date.now()
+
+  for (const entry of store) {
+    // mark for deletion entries that have timed out
+    if (now - entry[1][0] > entry[1][1]) {
```

Comment:
can you store the data in an object instead of an array, so we have named properties here? that would make it more clear what the data is, compared to just accessing the data by index (for example, I can't tell what entry[1][1] actually is)